UnicornEngine/video/include/unicorn/video/vulkan/Renderer.hpp

```diff
@@ -26,56 +21,83 @@ class Window;
 
 namespace video
 {
+namespace vulkan
+{
+/**
+ * @brief Struct for easy check of required queue family indices
+ */
 struct QueueFamilyIndices
 {
-    int graphicsFamily = -1;
-    int presentFamily = -1;
-
-    bool IsComplete() const
-    {
-        return graphicsFamily >= 0 && presentFamily >= 0;
-    }
+    int32_t graphicsFamily = -1;
+    int32_t presentFamily = -1;
+
+    /**
+     * @brief Check if all needed family indices are exists.
+     * @return true if graphics and present families exists and false if not
```

Comment:
Do not describe all checked parameters, describe what it means to be complete - `all required features are available` or something like that